### ISSUE TITLE
Suggestions on instrumenting travisCI with parallel stages/jobs

### ISSUE THREAD

**Issue by edulop91** [honeycombio/buildevents#51]
Hi, I was wondering if you had some suggestions on how to instrument a travisCI configuration that has multiple stages/jobs that are running in parallel. The way I'm currently thinking makes each job its own trace.

**Comment by thomaspoignant** [comment]
Yes, I have the exact same question.
If you have any input on how to do that it would be awsome.

**Comment by pkanal** [comment]
Hello,
We are trimming our OSS backlog. We will be closing this issue as it is a low priority for us to fix. It is unlikely that we'll ever get to it, and so we'd like to set expectations accordingly.

If this issue is important to you, please feel free to ping here and we can discuss/re-open.
